app/models/item_change_set.rb

```diff
@@ -31,8 +34,9 @@ def id
   # When the object is initialized, copy the properties from the cocina model to the form:
   def setup_properties!(_options)
     if model.identification
-      self.catkeys = Catkey.symphony_links(model)
-      self.refresh = Catkey.symphony_link_refresh(model)
+      self.catalog_record_ids = CatalogRecordId.links(model)
+      self.other_catalog_record_ids = CatalogRecordId.other_links(model)
```

Comment:
Added this to help persist Folio catalog links when in Symphony mode, and vice versa.